compiler/rustc_lint/messages.ftl

```diff
@@ -5,6 +5,10 @@ lint_array_into_iter =
     .use_explicit_into_iter_suggestion =
         or use `IntoIterator::into_iter(..)` instead of `.into_iter()` to explicitly iterate by value
 
+lint_async_fn_in_trait = usage of `async fn` in trait is discouraged because they do not automatically have auto trait bounds
+    .note = you can suppress this lint if you plan to use the trait locally, for concrete types, or do not care about auto traits like `Send` on the future
+    .suggestion = you can alternatively desugar the `async fn` and any add additional traits such as `Send` to the signature
```

Comment:
```
you can alternatively desugar the `async fn` manually by making it a regular, non-async fn and returning an `impl std::future::Future<Output = *return-type*>` and any add additional traits such as `Send` to the signature
```

Reply:
This is redundant given the contents of the suggestion showing you how to desugar immediately below this message

Reply:
Ahh, okay. Got it.